src/App.js

```diff
@@ -2,21 +2,38 @@ import React from 'react';
 import './App.css';
 import Nav from './Components/Nav';
 import Body from './Components/Body';
+import LogInForm from './Components/LogInForm';
 
 class App extends React.Component {
   constructor() {
     super();
     this.state = {
       movies: null,
       isLoading: false,
-      error: null
+      error: null,
+      // isLoggedIn: false,
+      // form: false,
+      logInMethod: this.logIn,
```

Comment:
is this form:false supposed to be commented out? are we able to render the login without it?

Reply:
Yes we are, was playing around and commenting stuff out to get rid of extraneous code